### PR TITLE
Feat/db connection

### PR DESCRIPTION
**Pull request by ngyngcphu**

### CODE REVIEW

File: app.vue [hunk 1]
```diff
@@ -1,6 +1,26 @@
+<script setup lang="ts">
+const message = apiRef({
+  route: '/api/auth/login',
+  method: 'post',
+  defaultValue: null
+});
```
Comment (huydo862003) [inline app.vue]: cool, lụm

File: app.vue [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+const { data } = await useFetch('/api/auth/login', {
+  method: 'post',
+  body: {
+    id: '2c2ec01b-ea45-4b5b-81ed-0bbbf2a06e28'
+  }
+});
+
+message.value = data.value;
```
Comment (huydo862003) [inline app.vue]: Something looks redundant :eyes:

**Review by huydo862003 — CHANGES_REQUESTED** [review]

File: app.vue (hunk 2 above)
Comment (ngyngcphu) [reply app.vue]: forgot push new code :>

File: app.vue [hunk 3]
```diff
@@ -1,6 +1,27 @@
...
+
+const { data } = await useFetch('/api/auth/login', {
+  method: 'post',
+  body: {
+    email: 'phu.nguyen2310@hcmut.edu.vn',
+    password: '123456789'
+  }
+});
+
+message.value = data.value;
```
Comment (huydo862003) [inline app.vue]: Can we merge these into one function call??

Comment (ngyngcphu) [reply app.vue]: > Can we merge these into one function call??

là seo

File: server/dtos/out/auth.dto.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { ObjectId } from "../common/objectId.dto";
```
Comment (huydo862003) [inline server/dtos/out/auth.dto.ts]: use `''` instead of `""` pls :pray: 
also set `alias` for nice import paths

File: app.vue (hunk 3 above)
Comment (huydo862003) [reply app.vue]: like this 
```
const message = apiRef({
  route: '/api/auth/login',
  method: 'post',
  defaultValue: await useFetch('/api/auth/login', {
  method: 'post',
  body: {
    email: 'phu.nguyen2310@hcmut.edu.vn',
    password: '123456789'
  }
  }). value;
});
```

**Pull request merged**